### PR TITLE
fixups and add travis

### PR DESCRIPTION
**Pull request by ahdinosaur**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+language: node_js
+node_js:
+  - "node"
```
Comment (wolfbiter) [inline .travis.yml:3]: how does this work? does it run both linting and and ava tests?

did you have to sign up for a service to get this to work?

File: channels/constants.js [hunk 2]
```diff
@@ -2,5 +2,5 @@ module.exports = {
   CHANNEL_TYPE_PRIMARY_TRACK: 'primary-track',
   CHANNEL_TYPE_SAMPLE_TRACK: 'sample-track',
   CHANNEL_TYPE_TRANSITION: 'transition',
-  CHANNEL_TYPE_MIX: 'mix',
+  CHANNEL_TYPE_MIX: 'mix'
```
Comment (wolfbiter) [inline channels/constants.js:5]: ah but i like trailing commas. do you feel strongly about this? why or why not?

**Comment by wolfbiter** [comment]
looks like all our ava tests are failing. do you know if your tests `nest` and `flatten` should be working?

**Comment by wolfbiter** [comment]
figured out the test issues and addressed them in https://github.com/linxmix/linx-electron/pull/6

File: channels/constants.js (hunk 2 above)
Comment (wolfbiter) [reply channels/constants.js:5]: i see, so this is `standardjs`'s thing? i read about their anti-bikeshed ethos. just want to confirm this is that

File: .travis.yml (hunk 1 above)
Comment (ahdinosaur) [reply .travis.yml:3]: it runs `npm test` which runs both (see scripts in `package.json`). it's free for open source.

File: channels/constants.js (hunk 2 above)
Comment (ahdinosaur) [reply channels/constants.js:5]: ah yes. i'm actually with you on trailing commas, but for now i do `standard`. it's possible to use [`standard-engine`](https://github.com/flet/standard-engine) to make your own.

**Pull request merged**